Exercises/multi_gpu_cuda/Begin/CMakeLists.txt

```
@@ -1,7 +1,13 @@
 cmake_minimum_required(VERSION 3.16)
 project(KokkosTutorialMultiGpuCuda)
-include(../../common.cmake)
-  
-add_executable(ExerciseMultiGPU multi_gpu_cuda.cpp)
-target_link_libraries(ExerciseMultiGPU Kokkos::kokkos)
 
+# Add a custom module path for find_package(Kokkos)
+include(../../SetUpKokkos.cmake)
+
+if (NOT Kokkos_ENABLE_CUDA)
+    message(WARNING "This example requires CUDA, enable with -DKokkos_ENABLE_CUDA=ON")
+    return()
```

```suggestion
    message(FATAL_ERROR "This example requires CUDA, enable with -DKokkos_ENABLE_CUDA=ON")
```

The code will not compile without Cuda.